File: PatientsServer/src/main/java/DTOs/GetPrescriptionsDto.java
package DTOs;

public class GetPrescriptionsDto
{
  public int id;
  public String diagnosis;
  public String medication;
  public String recommendations;
  public String date;
  public String time;
  public String patientcpr;
  public String doctorid;
  public String doctorname;
  public String doctorsurname;

  public GetPrescriptionsDto(int id, String diagnosis, String medication, String recommendations, String date, String time, String patientcpr, String doctorid, String doctorname, String doctorsurname) {
    this.id = id;
    this.diagnosis = diagnosis;
    this.medication = medication;
    this.recommendations = recommendations;
    this.date = date;
    this.time = time;
    this.patientcpr = patientcpr;
    this.doctorid = doctorid;
    this.doctorname = doctorname;
    this.doctorsurname = doctorsurname;
  }

  public int getId() {
    return id;
  }

  public String getDiagnosis() {
    return diagnosis;
  }

  public String getMedication() {
    return medication;
  }

  public String getRecommendations() {
    return recommendations;
  }

  public String getDate() {
    return date;
  }


  public String getTime() {
    return time;
  }

  public String getPatientCpr() {
    return patientcpr;
  }


  public String getDoctorId() {
    return doctorid;
  }

  public String getDoctorname()
  {
    return doctorname;
  }

  public String getDoctorsurname()
  {
    return doctorsurname;
  }
}

File: Patient/Patient/DTOs/GetPrescriptionsDto.cs
namespace Patient.DTOs;

public class GetPrescriptionsDto
{
    public int id { get; set; }
    public string diagnosis { get; set; }
    public string medication { get; set; }
    public string recommendations { get; set; }
    public string date { get; set; }
    public string time { get; set; }
    public string patientcpr { get; set; }
    public string doctorid { get; set; }
    
    public string doctorname { get; set; }
    public string doctorsurname { get; set; }

    public GetPrescriptionsDto(int id, string diagnosis, string medication, string recommendations, string date, string time, string patientcpr, string doctorid, string doctorname, string doctorsurname)
    {
        this.id = id;
        this.diagnosis = diagnosis;
        this.medication = medication;
        this.recommendations = recommendations;
        this.date = date;
        this.time = time;
        this.patientcpr = patientcpr;
        this.doctorid = doctorid;
        this.doctorname = doctorname;
        this.doctorsurname = doctorsurname;
    }

}